core/src/main/java/net/sourceforge/jnlp/util/WindowsDesktopEntry.java

```diff
@@ -171,10 +182,13 @@ private void manageShortcutList(ManageMode mode, String path) throws IOException
                 }
             }
             if (fAdd) {
-                LOG.debug("Adding Shortcut to list = ", sItem);
-                String scInfo = file.getFileLocation().toString() + ",";
-                scInfo += path + "\r\n";
-                Files.write(getWindowsShortcutsFile().toPath(), scInfo.getBytes(), StandardOpenOption.APPEND);
+                LOG.debug("Adding Shortcut to list = {}", sItem);
+                LOG.debug("Default encoding is {}", Charset.defaultCharset().name());
+                StringBuilder stringBuilder = new StringBuilder().append(file.getFileLocation().toString())
```

Comment:
With modern java versions there is no longer a performance gain of StringBuilder over simple string concatenation.

Reply:
changed and committed